src/consensus.rs

```diff
@@ -231,23 +231,20 @@ impl <L, M> Consensus<L, M> where L: Log, M: StateMachine {
                         self.persistent_log.entry(LogIndex(prev_log_index)).unwrap().0
                     };
 
-                let mut message = MallocMessageBuilder::new_default();
```

Comment:
Is this just cleanup, or is it related to the bug? (the former is fine, I just want to make sure I understand what's going on)